examples/question-answering/fsdp_config.json

```diff
@@ -8,5 +8,6 @@
     "fsdp_sync_module_states": true,
     "fsdp_use_orig_params": true,
     "transformer_layer_cls_to_wrap": "BertLayer",
-    "fsdp_activation_checkpointing": false
+    "fsdp_activation_checkpointing": false,
```

Comment:
Please also update peft version in requirement file (ignore if it is already done)

Reply:
same as above comment.